themes/default/content/blog/pulumi-release-notes-68/index.md

```diff
@@ -0,0 +1,128 @@
+- Pulumi CLI and core technologies
+  - [Public preview of Update Plans](#public-preview-of-update-plans)
+  - [Disable default provider](#disable-default-provider)
+  - [`pulumi up --target` wildcard URN support](#pulumi-up---target-wildcard-urn-support)
+- Pulumi Service
+  - [Sign-up and sign-in experience revamp](#sign-up-and-sign-up-experience-revamp)
+  - [Show resource URN on resource detail page](#show-resource-urn-on-resource-detail-page)
+
+<!--more-->
+
+## Cloud Providers and Packages
+
+### Helm Release for Kubernetes GA
+
+Back in September 2021 we announced public preview for the Helm Release resource in Pulumi’s Kubernetes provider. Over the last few months, we have had a very encouraging uptake in usage and several meaningful discussions with users in the community that have helped shape improvements to this resource. Thanks to this collaboration, we have been able to make the Helm Release resource GA (generally available) in v3.15.0 of the Pulumi Kubernetes Provider and SDK, in all Pulumi supported languages. We are excited to offer yet another tool to Pulumi users to effectively manage their Kubernetes footprint.
+
+
+Learn more in our [Helm release blog post]({{< relref "/blog/helm-release-resource-for-kubernetes-generally-available" >}}) and these GitHub issues:
+- [Honor namespace specification for all resources installed by Helm Release](https://github.com/pulumi/pulumi-kubernetes/pull/1747)
+- [Make values optional](https://github.com/pulumi/pulumi-kubernetes/pull/1761)
+- [Make RepositoryOpts optional](https://github.com/pulumi/pulumi-kubernetes/pull/1806)
+- [Support local charts](https://github.com/pulumi/pulumi-kubernetes/pull/1809)
+
+
+### Consider default org when running commands that accept stack names
+Now that we have added the ability to take [advantage of a default org](https://github.com/pulumi/pulumi/pull/8352), we have added functionality to take into account where there are abilities to pass a stack name as part of a command, such as `pulumi up -s <stackname>` or as part of a `stack select` command. [Learn more in the GitHub issue](https://github.com/pulumi/pulumi/issues/8409).
+
+## Pulumi CLI and core technologies
+
+### Public preview of Update Plans
+
+Pulumi’s previews are an important part of any workflow where you want to see the changes that will be made to your infrastructure before actually making the changes (with `pulumi up`). However, today there is no guarantee that the `pulumi up` operation will do only what was previewed; if the program, or your infrastructure, changes between the preview and the update, the update might make additional changes to bring your infrastructure back in line with what’s defined in your program. We’ve heard from many of you that you need a strong guarantee about exactly which changes an update will make to your infrastructure, especially in critical and production environments.
```

Comment:
I might tone down the “there was no guarantee”. Might phrase as “previously , there were cases where…” or similar.